### DISCUSSION TITLE
XF5 Prev4: PopUp with ListView / CollectionView not working

### DISCUSSION THREAD

**Discussion by HarryLobster** [PrismLibrary/Prism#2266] [Xamarin.Forms]
I have successfully implemented the IDialogService into my App to have modal Dialogs. 
Now I wanted to develop a DialogPicker that includes a ListView or a CollectionView for the Items to pick. 
But when I do that, the Page is showing up but contains nothing at all, I mean not even the Label.

This does not work:

```
<Label Text="Test"/>
<CollectionView></CollectionView>
```
This works:

`<Label Text="Test"/>`

I don´t know yet if it is a bug, a missing feature or myself. I wanted to start with a simple question...

**Comment by dansiegel** [comment]
Generally with issues like this it's best to have a sample app that people can look at to run and help you see what you're doing wrong or identify an actual bug if one exists.

**Comment by Tum4ik** [comment]
I have the same problem. I'll try to provide the example.

**Comment by Tum4ik** [comment]
Here we go: https://github.com/Tum4ik/prism-dialogservice-problem
When you start the project and press the "Test Dialog" button you will notice there are two items in the list view, but they are empty.

![image](https://user-images.githubusercontent.com/21317790/114315564-3b305900-9b08-11eb-9657-c9d25447cbb5.png)

However, you can force it to show the text via "Hot reload":
in the ListSelectionDialog.xaml file change the line during debug session with hot reload enabled
```xaml
<!-- From --><TextCell Text="{Binding}" TextColor="Black"/>
<!-- To --><TextCell Text="{Binding .}" TextColor="Black"/>
```
You will see the text appears in the list view items.

![image](https://user-images.githubusercontent.com/21317790/114315689-b134c000-9b08-11eb-8e4b-6dd059865ad6.png)

**Comment by Tum4ik** [comment]
So, can anybody please say what it is? Should we create a bug may be? @dansiegel @brianlagunas

**Comment by Tum4ik** [comment]
Wow, got it working via changing
```xaml
<!-- From --><TextCell Text="{Binding}" TextColor="Black"/>
<!-- To --><TextCell Text="{Binding BindingContext, Source={RelativeSource Self}}" TextColor="Black"/>
```
So strange...

Also I have a real project and now the data template looks like:
```xaml
<DataTemplate>
  <ViewCell x:Name="ViewCell">
    <Grid ColumnDefinitions="Auto, *">
      <skia:SKCanvasView Grid.Column="0" />
      <Label Grid.Column="1"
             Text="{Binding BindingContext, Source={x:Reference ViewCell}, Converter={StaticResource SupportedLocalizationToNativeNameConverter}}"
             HorizontalTextAlignment="End"
             VerticalTextAlignment="Center" />
    </Grid>
  </ViewCell>
</DataTemplate>

```

**Comment by Tum4ik** [comment]
It seems to me that the binding inside the data template at the initialization is connected to the dialog binding context instead of view cell binding context.
